hedera-mirror-rest/viewmodel/customFeeLimitViewModel.js

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0
+
+class CustomFeeLimitViewModel {
+  /**
+   * Formats the CustomFeeLimit data for API response.
+   * @param {CustomFeeLimits} customFeeLimits - Array of parsed CustomFeeLimit objects.
+   */
+  constructor(customFeeLimits) {
+    this.max_custom_fees = customFeeLimits.fees.flatMap((fee) =>
+      (fee.fixedFees ?? []).map((fixedFee) => ({
```

Comment:
because in protobuf definition, it's `fees`

```suggestion
      (fee.fees ?? []).map((fixedFee) => ({
```